crates/astria-bridge-withdrawer/src/config.rs

```diff
@@ -18,8 +18,6 @@ pub struct Config {
     pub sequencer_key_path: String,
     // The fee asset denomination to use for the bridge account's transactions.
     pub fee_asset_denomination: asset::Denom,
-    // The minimum expected balance of the fee asset in the bridge account.
-    pub min_expected_fee_asset_balance: u64,
```

Comment:
Mentioned in the example config, reproduced here:
Removing this is a breaking change but it's not marked in the PR title nor explained in the PR message.